components/InventoryItemListItem/index.tsx

```diff
@@ -75,44 +75,35 @@ export default function InventoryItemListItem({
   }
 
   return (
-    <TableRow sx={{ display: 'flex' }}>
+    <TableRow>
       <TableCell
         sx={{
-          display: 'flex',
           alignItems: 'center',
           justifyContent: 'center',
-          width: '100px',
           wordBreak: 'break-word',
         }}
       >
         {inventoryItem.itemDefinition.name}
       </TableCell>
       <TableCell
         sx={{
-          display: 'flex',
           alignItems: 'center',
           alignContent: 'center',
-          flexWrap: 'wrap',
           gap: '0.25rem',
-          width: '400px',
         }}
       >
         {renderAttributeChips(inventoryItem)}
```

Comment:
This renders a little bit wonky, see if you can fix that up. Might resort to going back to a flex box for that, not sure. Play around and see what works

![image](https://user-images.githubusercontent.com/71574118/229696850-2e2771e2-8948-4200-997a-d1eeceee88a6.png)